tooling/create-tauri-app/CHANGELOG.md

```diff
@@ -1,10 +1,5 @@
 # Changelog
 
-## \[1.0.0-beta-rc.1]
```

Comment:
Is it OK to lose this?

Reply:
This section was incorrectly added. See line 13 / 8 below; it already exists. It _should_ have been `-beta-rc.3`.

Reply:
But that `-beta-rc.3` never published because the version was wrong.